roles/static_inventory/templates/inventory.j2

```diff
@@ -44,6 +44,7 @@ masters.{{ stack_name }}
 
 [etcd:children]
 etcd.{{ stack_name }}
+{% if groups['etcd']|length == 0 %}masters.{{ stack_name }}{% endif %}
```

Comment:
In my testing, the `groups` didn't even have `etcd` so this step failed on undefined variable.

Changing it to this seems to work:

    {% if 'etcd' not in groups or groups['etcd']|length == 0 %}masters.{{ stack_name }}{% endif %}